src/fu-debug.c

```diff
@@ -239,8 +260,11 @@ fu_debug_pre_parse_hook(GOptionContext *context, GOptionGroup *group, gpointer d
 	     "DOMAIN"},
 	    {NULL}};
 
-	/* add main entry */
-	g_option_context_add_main_entries(context, main_entries, NULL);
+	/* set from FuConfig */
+	if (g_strcmp0(g_getenv("FWUPD_VERBOSE"), "*") == 0)
```

Comment:
I'd actually question if `VerboseDomains=` is useful.  It's a glorified grep.  Maybe we should just axe it all together?

Reply:
I've used it for profiling in the past -- i.e. not wasting cycles syncing everything, just the profiling.

Reply:
I'll re-profile and see if it makes it on the profile graph.